curator/Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM centos:centos7
+FROM rhel7.3:7.3-released
 
 MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 
 # by default, run the curator cron job at midnight UTC every night, and delete
-# indices older than 31 days old
+# indices older than 30 days old
```

Comment:
This is what is in the 3.5 branch of ose... it may be we need to update it to be 31 like origin.  I seem to recall an issue where we might have been missing deletes where we changed it to 31.  @richm ??